website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -0,0 +1,94 @@
+# `invoke`
+
+```python
+def invoke(
+    contract_address: int,
+    function_name: str,
+    inputs: list[int] | dict[str, Any] | None = None,
+    *,
+    config: SignedCheatcodeConfig | None = None,
+):
+```
+
+`SignedCheatcodeConfig` stores configuration used in cheatcodes that can be signed (invoke, deploy).
+
+This cheatcode invokes a StarkNet contract, with possible state changes. Can be useful for initializing proxies, etc.
+
+Auto-fee estimation is supported, and `starknet.py`'s estimation logic is used - see [starknet.py docs](https://starknetpy.readthedocs.io/en/latest/guide.html#automatic-fee-estimation).
+
+
+:::tip
+You can provide `inputs` as a dictionary to use [data transformer](./README.md#data-transformer).
+:::
+
+## Fees
+`SignedCheatcodeConfig` stores configuration used in cheatcodes that can be signed.
+It's an extension of [CheatcodeNetworkConfig](../03-network-config.md), so it's properties are applicable here as well.
+
+```python
+Wei = int
+class SignedCheatcodeConfig(CheatcodeNetworkConfig):
+    max_fee: Wei | None
+    auto_estimate_fee: int    
+```
+
+Either `max_fee` (in Wei) or `auto_estimate_fee` is required.
+We recommend using `max_fee` to avoid unexpected network costs.
+The config object is passed as a python dictionary.
+This config object also contains properties of `CheatcodeNetworkConfig`, see configuration options in the [related docs](../03-network-config.md).
+
+Wallet used for providing the fee is global, and is provided with signing arguments, as described [here](../01-cli.md#signing-a-declaration).
```

Comment:
Maybe `SignedCheatcodeConfig` section should be in the `03-network-config.md`. This config is going to be used by many cheatcodes.

Reply:
I thought about it, and i would hold with doing this now

Reply:
Lets do it when another one actually uses it

Reply:
At the other hand, relocating sections later could break links to that section.

Reply:
We're not going to relocate anything here, since there is no separate section for that